packages/adblocker-webextension/test/index.test.ts

```diff
@@ -125,3 +131,209 @@ describe('#fromWebRequestDetails', () => {
     });
   });
 });
+
+describe('html-filtering', () => {
+  context('#shouldApplyReplaceSelectors', () => {
+    function createWebRequestDetails({
+      responseType,
+      headers,
+    }: {
+      responseType: WebRequest.ResourceType;
+      headers: Record<string, string>;
+    }) {
+      const url = 'https://foo.com/';
+      const requestId = 'req-01';
+      const tabId = 1;
```

Comment:
all those can be inlined with the details object literal

Reply:
how about 
```js
const createWebRequestDetails = ({ type, headers }) => ({
  requestId: "req-01",
  tabId: 1,
  url: "https://foo.com/",
  type,
  responseHeaders: Object.entries(headers).map(([name, value]) => ({
    name,
    value,
  })),
});
```

Reply:
Updated in [`9991a4a` (#14)](https://github.com/seia-soto/adblocker/pull/14/commits/9991a4a7979f149105991bd4b199be1250a86d41)